go-apps/meep-ams/server/ams.go

```diff
@@ -760,11 +775,12 @@ func checkAdjAppInfoNotificationRegisteredSubscriptions(appNames []string) {
 
 func checkPeriodicTrigger() {
 
+	// PATCH: Update appStore with DB contents periodically.
```

Comment:
Unclear why the PATCH is needed - should be clarified in comment or removed if not needed.

Reply:
Fixed in PR #327